### PR TITLE
SPOI-12278 Config changes to app-templates.

### PR DESCRIPTION
**Pull request by deepak-narkhede**

### CODE REVIEW

**Comment by deepak-narkhede** [comment]
@yogidevendra. Please review.

File: app-templates/hdfs-to-hdfs-filter-transform/pom.xml [hunk 1]
```diff
@@ -10,6 +10,9 @@
   <name>HDFS to HDFS Filter Transform App</name>
   <version>0.10.0</version>
   <description>Ingest and backup hadoop HDFS data as lines from one cluster to another in a fault tolerant way.</description>
+  <properties>
+    <apex.apppackage.requiresRTSVersion>3.9.0</apex.apppackage.requiresRTSVersion>
```
Comment (yogidevendra) [inline app-templates/hdfs-to-hdfs-filter-transform/pom.xml]: Please add tags, longDescription as well for this.

Comment (deepak-narkhede) [inline app-templates/hdfs-to-hdfs-filter-transform/pom.xml]: Done.

**Comment by deepak-narkhede** [comment]
@yogidevendra . Couldn't find missing dependency for janino, supercsv or json-validator. Rest comments are incorporated.

**Pull request merged**